src/components/Dropdown/Dropdown.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Dropdown.scss';
+
+class Dropdown extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      expanded: false,
+      selectedID: 0,
+      options: props.options,
+    };
+    this.toggleDropdown = this.toggleDropdown.bind(this);
+    this.selectOption = this.selectOption.bind(this);
+  }
+
+  selectOption(e) {
+    const option = e.target;
+    this.setState({
+      selectedID: option.getAttribute('selectednum'),
+    });
+  }
+
+  toggleDropdown() {
+    this.setState( {
+      expanded: !this.state.expanded,
+    });
+  }
+
+  renderMenuItems() {
+    return this.state.options.map(({ value, label }, i) => (
+      <li
+        className="dropdown__items__element"
+        key={value}
+        selectednum={i}
+        value={value}
+        onClick={this.selectOption}
+      >
+        {label}
+      </li>
+    ));
+  }
+
+  render() {
+    const iconName = `caret ${this.state.expanded ? 'up' : 'down'} icon`;
+    const { expanded, selectedID } = this.state;
```

Comment:
Please change order: first, destructure and then use expanded inside iconName literal